src/components/dashboard/SafeAppsDashboardSection/SafeAppsDashboardSection.tsx

```diff
@@ -6,12 +6,15 @@ import Button from '@mui/material/Button'
 
 import { WidgetContainer } from '../styled'
 import { useSafeApps } from '@/hooks/safe-apps/useSafeApps'
-import { AppCard, AppCardContainer } from '@/components/safe-apps/AppCard'
+import useSafeAppPreviewDrawer from '@/hooks/safe-apps/useSafeAppPreviewDrawer'
+import SafeAppPreviewDrawer from '@/components/safe-apps/SafeAppPreviewDrawer/SafeAppPreviewDrawer'
```

Comment:
why the tautology? `SafeAppPreviewDrawer/SafeAppPreviewDrawer`

Reply:
same as `SafeAppIconCard` component

Reply:
done! renamed from `SafeAppPreviewDrawer.tsx` to `index.tsx`